pkg/service/test/test.go

```diff
@@ -152,7 +161,8 @@ func (t *tester) Test(path, testReportPath string, appCmd string, testsets []str
 		}
 	}
 	t.logger.Info("test run completed", zap.Bool("passed overall", result))
-
+	//Call the telemetry events.
+	tele.Testrun(resultForTele[0], resultForTele[1])
```

Comment:
Check if you are firing this event in all required places

Reply:
done